.ci/travis.sh

```diff
@@ -5,6 +5,7 @@ set -euo pipefail
 case "$1" in
 
 install)
+  mvn -e install:install-file
```

Comment:
I do not understand this hack.
Please do:
```
git clone .....medata.git
CD metadata
Mvn install
CD ../

```

Reply:
Done